packages/katello/katello/hostname-change.rb

```diff
@@ -255,7 +255,7 @@ def run
         STDOUT.puts "\nChecking custom certificates"
         fail_if_file_not_found([@options[:custom_cert], @options[:custom_key], @options[:custom_cert_req]])
```

Comment:
You can probably remove it here as well so it doesn't fail if the file doesn't exist.
```suggestion
         fail_if_file_not_found([@options[:custom_cert], @options[:custom_key]])
```